drivers/serial/uart_stm32.c

```diff
@@ -1446,6 +1447,9 @@ static int uart_stm32_pm_control(const struct device *dev,
 		LL_USART_ClearFlag_ORE(UartInstance);
 		/* Leave UartInstance unchanged */
 		break;
+	case PM_DEVICE_ACTION_RESUME:
+		/* Leave UartInstance ready to Tx/Rx */
```

Comment:
```suggestion
		/* Nothing to be done, UART is not in low power mode */
```

Reply:
After checking, this should not be required, since the system doesn't check for any sort of error when resuming (should likely be fixed...) https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/pm/device.c#L72